packages/static-build/src/utils/gatsby.ts

```diff
@@ -303,6 +311,36 @@ module.exports = gatsbyNode;
   );
 }
 
+export async function cleanupGatsbyFiles(dir: string) {
+  const backup = '.__vercel_builder_backup__';
+  const fileEndings = ['.js', '.ts', '.mjs'];
+
+  for (const fileName of [GATSBY_CONFIG_FILE, GATSBY_NODE_FILE]) {
+    for (const fileEnding of fileEndings) {
```

Comment:
**question (non-blocking):** Should this try-catch around each operation so the rest of the operations can continue if one errored?